conf/app.routes

```diff
@@ -385,3 +385,11 @@ POST        /update-disclosure/reference-number
 
 GET         /update-disclosure/need-to-upload                                                  controllers.MoreDocumentationController.onLoad
 POST        /update-disclosure/need-to-upload                                                  controllers.MoreDocumentationController.onSubmit
+
+GET         /update-disclosure/upload-file                                                     controllers.UploadSupportingDocumentationController.onLoad
+GET         /update-disclosure/upload-file/upscan-response                                     controllers.UploadSupportingDocumentationController.upscanResponseHandler(key: Option[String] ?= None, errorCode: Option[String] ?= None, errorMessage: Option[String] ?= None, errorResource: Option[String] ?= None, errorRequestId: Option[String] ?= None)
+GET         /update-disclosure/upload-file/polling                                             controllers.UploadSupportingDocumentationController.uploadProgress(key: String)
+GET         /update-disclosure/upload-another-file                                             controllers.UploadSupportingDocumentationSummaryController.onLoad
```

Comment:
This url is not the same as the one in the errors/urls spreadsheet, however I prefer it as it is the same as the one for create case. Can you have a chat with Nikki and make sure she is ok that we have changed it to be consistent with create case?